src/manifest.json

```diff
@@ -59,22 +63,25 @@
     "https://consumerapi.ghosterystage.com/*",
     "https://ghosterysearch.com/*",
     "https://staging.ghosterysearch.com/*",
+    "https://glowstery.com/*",
+    "https://staging.glowstery.com/*",
     "http://localhost/*"
   ],
   "icons": {
-    "16": "favicon.ico"
+    "48": "icon.svg",
+    "96": "icon.svg"
   },
   "web_accessible_resources": [
-    "favicon.ico"
+    "icon.svg"
   ],
   "chrome_settings_overrides" : {
     "search_provider": {
-      "name": "Ghostery Search",
-      "search_url": "https://ghosterysearch.com/search?q={searchTerms}",
-      "keyword": "@ghostery",
-      "favicon_url": "https://ghosterysearch.com/img/favicon-32x32.png",
+      "name": "Ghostery Glow",
+      "search_url": "https://glowstery.com/search?q={searchTerms}",
+      "keyword": "@glowstery",
```

Comment:
Should we really change this keyword? Is it used to trigger custom search from URL bar?

Reply:
yeah, and only in the case Glow is not default. but yeah, valid point. I will rollback to Ghostery.